store/storetest/post_store.go

```diff
@@ -698,13 +699,13 @@ func testPostStoreGetPostsWithDetails(t *testing.T, ss store.Store) {
 	_ = (<-ss.Post().Save(o6)).Data.(*model.Post)
 
 	// Should only be 6 since we hit the cache
-	r3 := (<-ss.Post().GetPosts(o1.ChannelId, 0, 30, true)).Data.(*model.PostList)
+	r3, _ := ss.Post().GetPosts(o1.ChannelId, 0, 30, true)
```

Comment:
By the way, you can insert the `require.Nil(t, err)` here and in the other similar places, if you want :)